app/Console/MarkPlaceAsDuplicateInBulkCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use Broadway\CommandHandling\CommandBus;
+use Broadway\EventHandling\EventListener;
+use CultuurNet\UDB3\Place\CannotMarkPlaceAsCanonical;
+use CultuurNet\UDB3\Place\CannotMarkPlaceAsDuplicate;
+use CultuurNet\UDB3\Place\Canonical\CanonicalService;
+use CultuurNet\UDB3\Place\Canonical\DuplicatePlaceRepository;
+use CultuurNet\UDB3\Place\Commands\MarkAsDuplicate;
+use Psr\Log\LoggerAwareInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class MarkPlaceAsDuplicateInBulkCommand extends AbstractCommand
+{
+    private EventListener $processManager;
+
+    private DuplicatePlaceRepository $duplicatePlaceRepository;
+
+    private CanonicalService $canonicalService;
+
+    public function __construct(
+        CommandBus $commandBus,
+        EventListener $processManager,
+        DuplicatePlaceRepository $duplicatePlaceRepository,
+        CanonicalService $canonicalService
+    ) {
+        parent::__construct($commandBus);
+        $this->processManager = $processManager;
+        $this->duplicatePlaceRepository = $duplicatePlaceRepository;
+        $this->canonicalService = $canonicalService;
+    }
+
+    public function configure(): void
+    {
+        $this->setName('place:mark-as-duplicate-bulk');
+        $this->setDescription('Marks multiple Places as duplicate of another one, implicitly making that one canonical');
+        $this->addOption(
+            'force',
+            null,
+            InputOption::VALUE_NONE,
+            'Skip confirmation.'
+        );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        $logger = new ConsoleLogger($output);
+
+        if ($this->processManager instanceof LoggerAwareInterface) {
+            $this->processManager->setLogger($logger);
+        }
+
+        $clusterIds = $this->duplicatePlaceRepository->getClusterIds();
+
+        foreach ($clusterIds as $clusterId) {
+            $placeCluster = $this->duplicatePlaceRepository->getCluster($clusterId);
+            $canonicalId = $this->canonicalService->getCanonical($placeCluster->getPlacesIds());
+            $duplicateIds = array_diff($placeCluster->getPlacesIds(), [$canonicalId]);
+
+            if (!$this->askConfirmation($input, $output, count($duplicateIds))) {
```

Comment:
I think asking this in a loop of clusters can be annoying 😅. As a work around you could just give the amount as clusters or add a extra method to get all places.